src/tests/rsa.rs

```diff
@@ -415,7 +415,7 @@ fn test_rsa_operations() {
     );
     assert_eq!(ret, CKR_OK);
 
-    #[cfg(not(feature = "fips"))]
+    #[cfg(all(not(feature = "fips"), feature = "dynamic"))]
```

Comment:
It is not clear to me the reasoning behind this change. Can you clarify? This was skipped in FIPS mode because it is PKCS#1.5 signature with SHA1 and raw. Now you exclude it also from the static builds, likely because of the new OpenSSL does not let it through? Would not it have the same meaning if you would keep only the second part? We do not support dynamic fips builds:
```suggestion
    #[cfg(feature = "dynamic")]
```
but I think its just matter of time when we will get openssl 3.5 in Fedora and we will need some other change of this condition to work  outside of the box ..

Reply:
In the static version I have (ossl 3.5 based) we have patches that completely disable SHA1 signatures, while the dynamic version (in CI) uses Fedora's Openssl version which does not disable SHA1 signatures.

Ideally we check what is available and then conditionally run tests, but I just wanted something quick here to get CI green, and defer the decision till later.

Note that these tests would fail on a RHEL system in all cases for example ...